server/utils/style_utils.ts

```diff
@@ -0,0 +1,6 @@
+import { prisma } from '~/server/prisma'
+
```

Comment:
seems there is no need for this file, you can just add that code where it's used. it's just one line.